packages/ripple-keypairs/test/codec.test.ts

```diff
@@ -1,9 +1,5 @@
-/* eslint-disable no-unused-expressions/no-unused-expressions */
-
-'use strict'
```

Comment:
Why was this removed?

Reply:
Eslint complains it isn't necessary in modules